### DISCUSSION TITLE
小写英文字母“t”和“f“连接的问题

### DISCUSSION THREAD

**Discussion by Monsterever** [atelier-anchor/smiley-sans#144] [Q&A] [answered]
英文字母"t"和"f"在使用中的连接不同，t在前时正常，如tfboys显示有间隔，若f在前，如drift，f和t会存在部分覆盖。
这点在有些例如手机APP网易云音乐的歌词中表现明显
![Snipaste_2024-02-26_22-42-06](https://github.com/atelier-anchor/smiley-sans/assets/68806975/7090f3ba-c5d4-4c96-bdc4-5862109bb0f2)
![Screenshot_20240226_215842_com netease cloudmusic](https://github.com/atelier-anchor/smiley-sans/assets/68806975/e56e75b9-5c86-4d3c-8a73-b2f9c8ad3311)

**Comment by stone-zeng** [comment, marked as answer]
设计即是如此，而在支持 opentype 特性的环境中实际上会调用另一个 `f.beforeit` 字符形：

<img width="285" alt="Screenshot 2024-02-27 at 10 55 07" src="https://github.com/atelier-anchor/smiley-sans/assets/18233088/0868a66a-cf45-4ba9-987d-4c5c3391081f">
<img width="285" alt="Screenshot 2024-02-27 at 10 55 14" src="https://github.com/atelier-anchor/smiley-sans/assets/18233088/d651cc61-f809-4f85-acb8-7014bd54887f">

**Reply by Monsterever** [reply]
感谢解答
